src/main/java/cubicchunks/world/column/Column.java

```diff
@@ -710,7 +714,8 @@ public boolean isTerrainPopulated() {
 		//with cubic chunks the whole column is never fully generated,
 		//So some heuristic is needed to tell vanilla is generator is populated here
 		//for now - tell it that it is if any cube is populated
-		return this.cubeMap.all().stream().anyMatch(c -> c.getGeneratorStage().isLastStage());
+		
+		return this.cubeMap.all().stream().anyMatch(c -> (c.getGeneratorStage() == GeneratorStage.LIVE));
```

Comment:
Why are you alternating between generationPipeline.isLastStage and comparing directly to LIVE?